features/admin.groups.v1/pages/groups.tsx

```diff
@@ -375,17 +372,15 @@ const GroupsPage: FunctionComponent<any> = (): ReactElement => {
                 onSortStrategyChange={ handleListSortingStrategyOnChange }
                 sortStrategy={ listSortingStrategy }
                 rightActionPanel={ (
-                    <RootOnlyComponent>
-                        <Dropdown
-                            data-testid="group-mgt-groups-list-stores-dropdown"
-                            selection
-                            options={ userStoreOptions && userStoreOptions }
-                            placeholder={ t("console:manage.features.groups.list.storeOptions") }
-                            onChange={ handleDomainChange }
-                            loading={ isUserStoresListRequestLoading || isUserStoreRequestLoading }
-                            defaultValue={ userstoresConfig.primaryUserstoreName }
-                        />
-                    </RootOnlyComponent>
+                    <Dropdown
+                        data-testid="group-mgt-groups-list-stores-dropdown"
+                        selection
+                        options={ userStoreOptions && userStoreOptions }
```

Comment:
Is this intentional?

Reply:
This is same as above. Enclosing `<RootOnlyComponent>` was removed around existing code.